condor/index.md

```diff
@@ -139,7 +139,7 @@ Among the improvements in VM 2.0 are:
 
 Subject to a governance vote by the Casper Network validators on behalf of their delegators, configuration changes in a future minor release of Casper 2.0 will activate a gas fee model known as **Fee Elimination**.
```

Comment:
```suggestion
Subject to a governance vote by the Casper Network validators on behalf of their delegators, configuration changes in a future minor release of Casper 2.0 could activate a gas fee model known as **Fee Elimination**.
```

Reply:
Thanks, @sacherjj , I've incorporated it.